src/txn/txn_ckpt.c

```diff
@@ -237,7 +237,6 @@ __checkpoint_data_source(WT_SESSION_IMPL *session, const char *cfg[])
 int
 __wt_checkpoint_list(WT_SESSION_IMPL *session, const char *cfg[])
 {
-	WT_DATA_HANDLE *saved_dhandle;
```

Comment:
This one is subtle, so I'd like to make sure I understand. It's no longer required because ```__wt_checkpoint_list``` is only ever called via ```__checkpoint_apply_all``` which in turn calls ```__wt_schema_worker```. The schema worker function now uses ```WT_SAVE_DHANDLE``` - so ```__wt_checkpoint_list``` no longer needs to save/restore.